app/main.py

```diff
@@ -2,0 +1,33 @@
+class Animal:
+
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+
+        # Додаємо екземпляр до класової змінної
+        if self.health > 0:
+            Animal.alive.append(self)
```

Comment:
Appending `self` to the class variable `Animal.alive` is correct if you want to track all living animals globally. Ensure this aligns with your design intentions.